cli/src/commands/config/mod.rs

```diff
@@ -81,19 +80,33 @@ impl ConfigLevelArgs {
         }
     }
 
-    fn edit_config_file(&self, command: &CommandHelper) -> Result<ConfigFile, CommandError> {
+    fn edit_config_file(
+        &self,
+        ui: &mut Ui,
+        command: &CommandHelper,
+    ) -> Result<ConfigFile, CommandError> {
         let config_env = command.config_env();
         let config = command.raw_config();
         let pick_one = |mut files: Vec<ConfigFile>, not_found_error: &str| {
             if files.len() > 1 {
-                // TODO: prompt or pick the last?
-                return Err(user_error(format!(
-                    "Cannot determine config file to edit:\n{}",
-                    files
-                        .iter()
-                        .map(|file| format!("  {}", file.path().display()))
-                        .join("\n")
-                )));
+                let mut choices = vec![];
+                let mut formatter = ui.stdout_formatter();
```

Comment:
nit: maybe we should use `stderr` because the prompt is now printed to stderr. (I think the existing caller is wrong.)